style.css

```diff
@@ -0,0 +1,107 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,300;0,400;1,300&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;1,100;1,300;1,400;1,500;1,700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+
+  /* overflow: hidden; */
+}
+
+.toolbar_container {
+  display: flex;
+  flex-direction: row;
+  background-color: #3c3a39;
+  max-width: 100%;
+  align-items: center;
+  justify-content: center;
+  align-content: center;
+  justify-items: center;
+  gap: 180px;
+  height: 17vh;
+  padding-bottom: 5rem;
+}
```

Comment:
- [ ] As suggested by the previous reviewer and mention in the project requirement kindly make sure that  Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) So Kindly Fix the alignment of your toolbar and as mentioned by the previous reviewer adjust `justify-content: space-between;`.
- The Figma Design Toolbar Look likes this : -
![toolbar](https://github.com/malodev1234/Portfolio_setup_and_mobile_First/assets/93598189/2322543d-aeb2-4619-a2be-5a2f43db3ad9)

- Your Toolbar section Look likes this : -

![toolbar](https://github.com/malodev1234/Portfolio_setup_and_mobile_First/assets/93598189/c71ce6ac-f451-4096-9b2e-93bb41af7edf)